docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -650,29 +1134,76 @@ For more details, refer to the GitHub issue [#360](https://github.com/yugabyte/y
+1. Disable triggers and foreign-key constraints on the source database to ensure that changes from the target YugabyteDB database can be imported correctly to the source database using the following PL/SQL commands on the source schema as a privileged user:<br>
 
-    ```sql
+{{< tabpane text=true >}}
+
+  {{% tab header="Oracle" %}}
+
+Use the following PL/SQL commands to disable triggers, and disable referential constraints on the source:
+
+  ```sql
     --disable triggers
-    BEGIN
-        FOR R IN (SELECT owner, object_name FROM all_objects WHERE owner=UPPER('<SCHEMA_NAME>') and object_type ='TABLE' MINUS SELECT owner, table_name from all_nested_tables where owner = UPPER('<SCHEMA_NAME>'))
-        LOOP
-          EXECUTE IMMEDIATE 'ALTER TABLE '||R.owner||'."'||R.object_name||'" DISABLE ALL TRIGGERS';
-        END LOOP;
-    END;
+  BEGIN
+      FOR R IN (SELECT owner, object_name FROM all_objects WHERE owner=UPPER('<SCHEMA_NAME>') and object_type ='TABLE' MINUS SELECT owner, table_name from all_nested_tables where owner = UPPER('<SCHEMA_NAME>'))
+      LOOP
+        EXECUTE IMMEDIATE 'ALTER TABLE '||R.owner||'."'||R.object_name||'" DISABLE ALL TRIGGERS';
+      END LOOP;
+  END;
     /
 
     --disable referential constraints
-    BEGIN
-        FOR c IN (SELECT table_name, constraint_name
-          FROM user_constraints
-          WHERE constraint_type IN ('R') AND OWNER = '<SCHEMA_NAME>')
-        LOOP
-          EXECUTE IMMEDIATE 'ALTER TABLE ' || c.table_name || ' DISABLE CONSTRAINT ' || c.constraint_name;
-        END LOOP;
-    END;
+  BEGIN
+      FOR c IN (SELECT table_name, constraint_name
+        FROM user_constraints
+        WHERE constraint_type IN ('R') AND OWNER = '<SCHEMA_NAME>')
+      LOOP
+        EXECUTE IMMEDIATE 'ALTER TABLE ' || c.table_name || ' DISABLE CONSTRAINT ' || c.constraint_name;
+      END LOOP;
+  END;
     /
-    ```
+  ```
+
+  {{% /tab %}}
+
+  {{% tab header="PostgreSQL" %}}
+
+  Use the following PL/SQL commands to disable triggers, and drop foreign-key constraints on the source:
+
+  ```sql
+  --disable triggers
+  DO $$
+  DECLARE
+      r RECORD;
+  BEGIN
+      FOR r IN
+          SELECT table_schema, '"' || table_name || '"' AS t_name
+          FROM information_schema.tables
+          WHERE table_type = 'BASE TABLE'
+          AND table_schema IN (<SCHEMA_LIST>)
+      LOOP
+          EXECUTE 'ALTER TABLE ' || r.table_schema || '.' || r.t_name || ' DISABLE TRIGGER ALL';
+      END LOOP;
+  END $$;
+
+  --drop referential constraints
+  DO $$
+  DECLARE
+      fk RECORD;
+  BEGIN
+      FOR fk IN
+          SELECT conname, conrelid::regclass AS table_name
+          FROM pg_constraint
+          WHERE contype = 'f'
+      LOOP
+          EXECUTE 'ALTER TABLE ' || fk.table_name || ' DROP CONSTRAINT ' || fk.conname;
+      END LOOP;
+  END $$;
+  ```
```

Comment:
modify this second query to 
```sh
DO $$
DECLARE
    fk RECORD;
BEGIN
    FOR fk IN
        SELECT conname, conrelid::regclass AS table_name
        FROM pg_constraint
        JOIN pg_class ON conrelid = pg_class.oid
        JOIN pg_namespace ON pg_namespace.oid = pg_class.relnamespace
        WHERE contype = 'f'
        AND pg_namespace.nspname IN (<SCHEMA_LIST>)
    LOOP
        EXECUTE 'ALTER TABLE ' || fk.table_name || ' DROP CONSTRAINT ' || fk.conname;
    END LOOP;
END $$;

 --- SCHEMA_LIST used is a comma-separated list of schemas, for example, SCHEMA_LIST 'abc','public', 'xyz'.
```

and add this comment in the first query as well - 
```
--- SCHEMA_LIST used is a comma-separated list of schemas, for example, SCHEMA_LIST 'abc','public', 'xyz'.

```